docs/digger/installation/rhmap-properties.asciidoc

```diff
@@ -6,55 +6,56 @@ When using Digger from RHMAP, you can configure Digger by setting RHMAP properti
 |===
 | Property name | Environment variable | Description
 
-|buildfarm.version
```

Comment:
@laurafitzgerald This would have come from the property names in millicore. Must have kept them in as names. The `buildfarm.` prefix could probably be removed.